src/neuroconv/datainterfaces/ecephys/baselfpextractorinterface.py

```diff
@@ -38,7 +38,7 @@ def run_conversion(
         use_times: bool = False,  # To-do to remove, deprecation
         compression: Optional[str] = None,
         compression_opts: Optional[int] = None,
-        iterator_type: Optional[str] = "v2",
+        iterator_type: str = "v2",
```

Comment:
This is really optional, if you pass `None` then the electrical series is written without memory chunking:

https://github.com/catalystneuro/neuroconv/blob/3570c50a6aa2123ac43c9cbb88c4143d187a7d52/src/neuroconv/tools/spikeinterface/spikeinterface.py#L553-L555